lib/rex/post/meterpreter/extensions/stdapi/sys/registry.rb

```diff
@@ -166,6 +165,24 @@ def Registry.enum_key(hkey)
     return keys
   end
 
+  def Registry.enum_key_direct(root_key, base_key, perm = KEY_READ)
+    request = Packet.create_request('stdapi_registry_enum_key_direct')
+    keys    = []
+
+    request.add_tlv(TLV_TYPE_ROOT_KEY, root_key)
+    request.add_tlv(TLV_TYPE_BASE_KEY, base_key)
+    request.add_tlv(TLV_TYPE_PERMISSION, perm)
+
+    response = client.send_request(request)
+
+    # Enumerate through all of the registry keys
+    response.each(TLV_TYPE_KEY_NAME) { |key_name|
+      keys << key_name.value
+    }
+
+    return keys
```

Comment:
You can omit the return keyword https://github.com/bbatsov/ruby-style-guide#no-explicit-return